composer.json

```diff
@@ -173,7 +173,7 @@
         "drupal/views_infinite_scroll": "^2.0",
         "drupal/views_taxonomy_term_name_depth": "^7.0",
         "drupal/webp": "^1.0@beta",
-        "drupal/xmlsitemap": "^2.0@beta",
```

Comment:
This is a beta release: https://www.drupal.org/project/xmlsitemap

`composer require 'drupal/xmlsitemap:^2.0@beta'`

Reply:
This should be fine.  It pulls in the beta with the ^2.0 constraint, and when it's eventually out of beta, it'll pull in the new version as well.